src/adaptor.py

```diff
@@ -190,8 +207,8 @@ def handler(json_request, outgoing):
     # we have a valid request :)
     LOG.info("valid request")
 
-    params = subdict(request, ['action', 'id', 'token', 'version'])
-    params['force'] = request.get('force') # optional value
+    params = subdict(request, ['action', 'id', 'token', 'version', 'force', 'validate-only'])
+    params = renkeys(params, [('validate-only', 'dry_run')])
```

Comment:
`validate-only` and `dry_run` are using two different conventions (`-` and `_`), which one are we targeting? `api-raml` usually goes for `camelCase`